docs/get-started/setting-up.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_label: 'Setting up'
+sidebar_position: 1
+---
+
+# Setting up
```

Comment:
To me, it would make the tutorial way easier to follow if we were to start with gently requesting the user to clone the repository and then consider that all the commands are run inside it. Otherwise, I have concerns about the usability of a tutorial requiring to punctually download raw assets from specific paths on GitHub while we could just simply clone the repository and work in it.

Don't you agree?

Reply:
Adapted tutorial in https://github.com/polyphene/docs.holium.org/pull/16/commits/ad139669837b5214463c75794aa3592cb09ee8e5